Resources/Prototypes/Entities/Objects/Specific/atmos.yml

```diff
@@ -32,3 +32,282 @@
     materialComposition:
       Steel: 400
       Glass: 100
+
+# atmos candles
+
+- type: entity
+  name: gas candle
+  description: A candle that emits hot gas when lit.
+  parent: Candle
+  id: CandleGasBase
+  abstract: true
+  components:
+  - type: Temperature
+  - type: Flammable
+    requireOxygen: false
+    emissiveGasMix: #  emits ~ mols * 300 if you collect all of it.
+      volume: 50
+      moles:
+      - 0 # oxygen
+      - 0 # nitrogen
+      - 0 # CO2
+      - 0 # Plasma
+      - 0 # Tritium
+      - 0 # Water vapor
+      - 0 # Ammonia
+      - 0 # N2O
+      - 0 # Frezon
+
+- type: entity
```

Comment:
Besides oxygen and nitrogen candles, should any of the other candles really exist/be in the game?

Reply:
In the back of my mind, I'm envisioning a way of producing these, but that's for later. For now its more a "why not?" kind of affair. If I do make a way to produce them I think a want to go a route that lets you make any mix you want, but Ill have to think about how to do that more. I was resisting the urge to do it with solutions for the time being because I think having to do the solution -> gasmix conversion would probably bloat these more than I like.

Reply:
Because I don't think it's prudent to add a bunch of entity prototypes that will never get used.

I didn't think too much about this -- but since there a species that require N2 instead of O2, could we have an "air candle" instead of separate candles that would satisfy both? (My guess is "no" since IIRC vox might be allergic to oxygen...)

Reply:
I still only think air/nitrogen/oxygen candles should exist. There are other ways to get the other gases. But I won't block over this. Up to you.